### DISCUSSION TITLE
Show markers per playlist

### DISCUSSION THREAD

**Discussion by gkertai** [videovillage/community#108] [Ideas]
Hi, Maybe this is possible, I just cannot figure out how… When I have a playlist loaded, which includes multiple files with markers of various colors, is there a way to list all my markers across the entire playlist? Or at least see in the playlist panel if a clip has any markers (of a particular color)?

BTW, the *Help* menu missing the standard *Search* field makes it harder to explore existing features. I was going to search for menu items related to *markers* or *playlist*, but there's no search field.

**Comment by gregcotten** [comment]
Oh this is so strange! I have a (hidden) debug menu that gets removed for users, and it's right before "Help". I think the system doesn't like me removing it or gets confused and actually ends up putting the help search in the Debug menu. I'm going to fix this!

**Comment by gkertai** [comment]
Thank you for looking into this! Does your answer imply that there is a menu option somewhere to make all playlist markers visible?

**Comment by gregcotten** [comment]
Oh sorry, missed the mountain for the forest. No not currently - but shouldn't be too hard to show the marker colors associated with a file in the playlist, at a minimum

**Comment by gkertai** [comment]
> [v1.5.6](https://videovillage.co/screen/releasenotes#1.5.6): Hopefully fixed an issue where the Help search bar was not appearing

Yes, the search bar is there. Thank you, Greg!

**Comment by gregcotten** [comment]
Excellent!
